src/components/bulk-email-tool/bulk-email-form/bulk-email-recipient/bulkEmailRecepient.scss

```diff
@@ -0,0 +1,9 @@
+.recipient-groups {
+  > div {
+    min-width: 50%;
+    padding-right: 0.5rem;
+    input {
+      padding: 0.5rem !important;
+    }
+  }
+}
```

Comment:
I am curious if I am reading this correctly. Still shaky on this and haven't seen this syntax too many times. This is overwriting the styling of the div(s?) in the `Form.CheckboxSet` component (sub-component?)? 

We do it this way with a selector (is that what the `>` is called?)? Is this targeting all divs in the component that uses this piece of CSS? Or just the first?

Reply:
to be honest, I have no clue the proper name 😅. the chevron denotes "all direct children of parent"; so in this case every `div` that is a child of this class gets the styles. so `recipient-groups > div` gets them but not `recipient-groups > div > div`.